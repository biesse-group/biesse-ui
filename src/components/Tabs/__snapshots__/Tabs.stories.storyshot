// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Layout/Tabs Default 1`] = `
.c1 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 40px;
  outline: 0;
  cursor: pointer;
  border: none;
  border-radius: 50px;
  padding: 0 25px;
  background-color: #F5F5F5;
  color: #6D6E70;
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 300;
  font-size: 16px;
  background-color: #6D6E70;
  color: #FFF;
}

.c2 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 40px;
  outline: 0;
  cursor: pointer;
  border: none;
  border-radius: 50px;
  padding: 0 25px;
  background-color: #F5F5F5;
  color: #6D6E70;
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 300;
  font-size: 16px;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  width: 100%;
  gap: 20px;
}

@media screen and (max-width:767px) {
  .c0 {
    overflow-x: auto;
  }
}

<div
  className="c0"
>
  <a
    href="./"
  >
    <button
      className="c1"
      onClick={[Function]}
    >
      Tutte
    </button>
  </a>
  <button
    className="c2"
    onClick={[Function]}
  >
    Corporate
  </button>
  <button
    className="c2"
    onClick={[Function]}
  >
    Prodotti
  </button>
  <button
    className="c2"
    onClick={[Function]}
  >
    Eventi
  </button>
  <button
    className="c2"
    onClick={[Function]}
  >
    Fiere
  </button>
</div>
`;
