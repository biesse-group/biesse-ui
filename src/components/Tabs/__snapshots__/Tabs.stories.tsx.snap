// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout/Tabs Default smoke-test 1`] = `
<div class="sc-gEvEer fXJvIt">
  <a href="./">
    <button class="sc-aXZVg fhASMz">
      Tutte
    </button>
  </a>
  <button class="sc-aXZVg ghVuiv">
    Corporate
  </button>
  <button class="sc-aXZVg ghVuiv">
    Prodotti
  </button>
  <button class="sc-aXZVg ghVuiv">
    Eventi
  </button>
  <button class="sc-aXZVg ghVuiv">
    Fiere
  </button>
</div>
`;
